src/index.html

```diff
@@ -4,9 +4,375 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="/images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link blue"
+              data-qa="nav-hover"
+            >
+             НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ГАДЖЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ФОТО</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="content">
+      <div class="content__product product">
+        <div class="product__card" data-qa="card">
```

Comment:
```suggestion
        <div class="card products__card" data-qa="card">
```
card должен быть независимым блоком, если нужно добавить геометрию, можно сделать это с помощью микса